src/client/components/app/Settings/ThemeBox.tsx

```diff
@@ -33,6 +33,7 @@ const ThemePreview = styled.div<{ selected?: boolean; themePallete: DefaultTheme
     background-color: ${(props) => props.themePallete.colors.primary};
     margin-right: 0.4rem;
   }
+
```

Comment:
👍

Reply:
Just vertical whitespace for readability